description.md

```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
+
+The media content type represents an easily digestible format for presenting information.
+As such, media is typically embbeded within the text that is destined reading, slides, demos, practice items and projects.
+
+When developing `media` items, it is important to follow the guidelines below:
```

Comment:
Why is media marked with backticks here but not elsewhere? I would remove backticks.